.github/workflows/test_on_trainium.yml

```diff
@@ -2,11 +2,13 @@ name: Optimum Neuron - Tranium-dependent Tests
 
 on:
   push:
-    branches:
-      - main
+    branches: [ main ]
+    paths:
+      - "optimum/**.py"
   pull_request:
-    branches:
-      - main
+    branches: [ main ]
+    paths:
+      - "optimum/**.py"
```

Comment:
Would it trigger the test if we change the workflow file?
Can we do like a filter saying "not docs" instead of saying what should trigger the CI?

Reply:
no but we can include the path as well